core/mule.c

```diff
@@ -89,12 +89,34 @@ void uwsgi_mule(int id) {
 }
 
 void uwsgi_mule_run() {
-	int id = uwsgi.muleid;
+        char* patch;
```

Comment:
yeah, done.  There was already spaces in `uwsgi_mule_run`, so I thought there is a tendency to use spaces